### PR TITLE
Deleteproject

### PR DESCRIPTION
**Pull request by bokova**
Addresses https://github.com/Learn-by-doing/bitstarter/issues/46
Incorporates @karliatto's add project functionality with a modified structure.

### CODE REVIEW

**Comment by bokova** [comment]
Note to self: add test

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+npm-debug.log
```
Comment (chill117) [inline .gitignore:2]: This could be its own pull request

File: views/projects-add.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="form-style-5">
```
Comment (chill117) [inline views/projects-add.html:1]: This file was already included (and merged) as part of a pull request by @karliatto

**Comment by chill117** [comment]
It looks like you might want to try a fresh branch from upstream/master and redo this pull request. It includes a lot of code (already merged) from @karliatto
